feat: Add PydanticAI template (#360)
apify/actor-templates: templates/python-pydanticai/README.md

## PydanticAI template

> Start a new [ai agent](https://blog.apify.com/what-are-ai-agents/) based project quickly and easily in Python with our PydanticAI project template. It provides a basic structure for the [Actor](https://apify.com/actors) with [Apify SDK](https://docs.apify.com/sdk/python/) and [PydanticAI](https://ai.pydantic.dev/) and allows you to easily add your own functionality.

Reviewer: capital "AI"

```suggestion
Start a new [AI agent](https://blog.apify.com/what-are-ai-agents/) based project quickly and easily in Python with our PydanticAI project template. It provides a basic structure for the [Actor](https://apify.com/actors) with [Apify SDK](https://docs.apify.com/sdk/python/) and [PydanticAI](https://ai.pydantic.dev/) and allows you to easily add your own functionality.
```

> Start a new [ai agent](https://blog.apify.com/what-are-ai-agents/) based project quickly and easily in Python with our PydanticAI project template. It provides a basic structure for the [Actor](https://apify.com/actors) with [Apify SDK](https://docs.apify.com/sdk/python/) and [PydanticAI](https://ai.pydantic.dev/) and allows you to easily add your own functionality.

Reviewer 2: ```suggestion
Start a new [AI agent](https://blog.apify.com/what-are-ai-agents/) based project in Python with our PydanticAI project template. It provides a basic structure for the [Actor](https://apify.com/actors) using the [Apify SDK](https://docs.apify.com/sdk/python/) and [PydanticAI](https://ai.pydantic.dev/), and allows you to add your own functionality with minimal setup.
```

Reviewer 2: Nit: when writing we are generally trying to avoid "easy", "quickly" etc. let users judge complexity.

## How it works

> Insert your own code to `async with Actor:` block. You can use the [Apify SDK](https://docs.apify.com/sdk/python/) with any other Python library. Add or modify the agent and tools in the [agents.py](.src/agents.py) file.

Reviewer: wrong relative file link (should be without the ".")

```suggestion
Insert your own code to `async with Actor:` block. You can use the [Apify SDK](https://docs.apify.com/sdk/python/) with any other Python library. Add or modify the agent and tools in the [agents.py](src/agents.py) file.
```

## Getting started

> For complete information [see this article](https://docs.apify.com/platform/actors/development#build-actor-locally). To run the actor use the following command:

Reviewer 2: ```suggestion
For complete information [see this article](https://docs.apify.com/platform/actors/development#build-actor-locally). To run the Actor use the following command:
```

```bash
apify run
```

## Deploy to Apify

### Connect Git repository to Apify

> If you've created a Git repository for the project, you can easily connect to Apify:

Reviewer 2: ```suggestion
If you've created a Git repository for the project, you can connect to Apify:
```

1. Go to [Actor creation page](https://console.apify.com/actors/new)
2. Click on **Link Git Repository** button

### Push project on your local machine to Apify

> You can also deploy the project on your local machine to Apify without the need for the Git repository.

Reviewer 2: ```suggestion
You can also deploy the project from your local machine to the Apify platform without the need for the Git repository.
```

To set up the PPE model for this Actor:
- **Configure the OpenAI API key environment variable**: provide your OpenAI API key to the `OPENAI_API_KEY` in the Actor's **Environment variables**.
- **Configure Pay Per Event**: establish the Pay Per Event pricing schema in the Actor's **Monetization settings**. First, set the **Pricing model** to `Pay per event` and add the schema. An example schema can be found in [pay_per_event.json](.actor/pay_per_event.json).

## Resources

To learn more about Apify, Actors and PydanticAI take a look at the following resources:

- [Apify SDK for Python documentation](https://docs.apify.com/sdk/python)
- [Apify Platform documentation](https://docs.apify.com/platform)
- [Join our developer community on Discord](https://discord.com/invite/jyEM2PRvMU)
- [AI agent architecture](https://blog.apify.com/ai-agent-architecture)
- [What are AI agents](https://blog.apify.com/what-are-ai-agents/)
- [How to build and monetize an AI agent on Apify](https://blog.apify.com/how-to-build-an-ai-agent/)
- [PydanticAI documentation](https://ai.pydantic.dev/) 

> - [PydanticAI on github](https://github.com/pydantic/pydantic-ai)

Reviewer: capital "GitHub"

```suggestion
- [PydanticAI on GitHub](https://github.com/pydantic/pydantic-ai)
```
